libkv/subscription.go

```diff
@@ -119,18 +107,7 @@ func (service Service) DeleteSubscription(space string, id subscription.ID) erro
 		return &subscription.ErrSubscriptionNotFound{ID: sub.ID}
 	}
 
-	if sub.Event == event.TypeHTTP {
-		err = service.deleteEndpoint(space, sub.Method, sub.Path)
```

Comment:
Why remove this?

Reply:
Some some time ago there was a concept of endpoints in EG. We removed it from Config API but it was still in the logic to prevent creating conflicting HTTP endpoint. Right now I migrated that logic to use subscriptions (which we need regardless) and this model and storing this in etcd is useless.

Reply:
I see.